.travis.yml

```diff
@@ -59,10 +59,12 @@ install:
         sed -i.bak "s/plugins *= *Cython\.Coverage//g" $cf;
       done
     fi
+  # temporarily use older version of pyarrow
+  - pip install "pyarrow<0.14"
+  - conda install --quiet --yes -n test -c conda-forge python=$PYTHON "tiledb-py>=0.4" pyarrow || true
```

Comment:
Is it duplicated that pyarrow is installed by pip and conda?

Reply:
I just installed a pyarrow==0.14 under linux and it works well with Mars. Maybe the true cause of the error needs investigation.

Reply:
I can reproduce [the error on Linux with python 36 and pyarrow 0.14](https://travis-ci.org/mars-project/mars/jobs/554169173), but haven't furtherly dug into the failure yet.

Reply:
The failure is caused by missing ``liblz4``. Already fixed.